### ISSUE TITLE
Error when searching for IP without prefix

### ISSUE THREAD

**Issue by Thetacz** [nautobot/nautobot#286]
### Environment
* Python version: 3.8.5
* Nautobot version: 1.0.0b3

When I use search field in the top right most corner (in navbar nex to user) and I search only by IP without prefix, I get the following error:
![image](https://user-images.githubusercontent.com/26114847/114363064-5470ee00-9b78-11eb-8235-1654a36985da.png)

Full trace:
```
Traceback (most recent call last):
  File "/opt/nautobot/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/opt/nautobot/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/django/views/generic/base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "/opt/nautobot/lib/python3.8/site-packages/nautobot/core/views/__init__.py", line 159, in get
    filtered_queryset = filterset({"q": form.cleaned_data["q"]}, queryset=queryset).qs
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filterset.py", line 243, in qs
    qs = self.filter_queryset(qs)
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filterset.py", line 230, in filter_queryset
    queryset = self.filters[name].filter(queryset, value)
  File "/opt/nautobot/lib/python3.8/site-packages/django_filters/filters.py", line 779, in __call__
    return self.method(qs, self.f.field_name, value)
  File "/opt/nautobot/lib/python3.8/site-packages/nautobot/ipam/filters.py", line 183, in search
    broadcast__gte=bytes(query.broadcast),

Exception Type: TypeError at /search/
Exception Value: cannot convert 'NoneType' object to bytes
```

Searching with prefix works:
![image](https://user-images.githubusercontent.com/26114847/114363397-a0bc2e00-9b78-11eb-8d2d-bb5fc2e13ce9.png)

### Steps to Reproduce
1. Search IP without prefix in the navbar box
![image](https://user-images.githubusercontent.com/26114847/114363465-b7628500-9b78-11eb-9cb7-83e1da8c4409.png)

### Expected Behavior
IP searched, results showed

### Observed Behavior
TypeError

**Comment by jathanism** [comment]
@Thetacz Thanks for the report. Looks like this will also be fixed by #281

**Comment by jathanism** [comment]
![Uploading Search_-_Nautobot.png…]()
